pkg/endpoint/getter/getter.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2016-2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package getter
+
+// EndpointGetter should be implemented by Endpoint so you don't
+// have to use pkg/endpoint if you only want to read Endpoint data in other package
+type EndpointGetter interface {
+	GetID() uint64
+	GetOpLabels() []string
+}
```

Comment:
I'm on the fence about this interface. It's nice since it makes it clear what we intend to do with the endpoint. On the other hand, we have `Endpoint.GetModel` that acts the same way, and we've been using it like that (I think. I didn't actually check). We only use the interface in one place and we could have passed in the two values we use. I imagine the idea was to allow us to add stuff later, but passing the model over would do the same thing (albeit less efficiently). How keen are you on the interface?